package.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "retrocast-docs-tooling",
+  "private": true,
+  "packageManager": "pnpm@11.1.2",
```

Biome (2.4.15)

[error] The key packageManager was already declared.

(lint/suspicious/noDuplicateObjectKeys)